### PR TITLE
Updating kibana6 image from for feature-es6x branch for origin-aggregated-logging

### PR DESCRIPTION
**Pull request by ewolinetz**
cc @jcantrill @richm @wking

### CODE REVIEW

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-es6x.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  to: logging-kibana6
+- context_dir: curator/
+  dockerfile_path: Dockerfile.centos7
+  from: base_centos
+  to: logging-curator5
+- context_dir: eventrouter/
+  dockerfile_path: Dockerfile.centos7
+  from: base_centos
+  to: logging-eventrouter
+- context_dir: rsyslog/
```
Comment (richm) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-es6x.yaml]: remove rsyslog

**Comment by ewolinetz** [comment]
it looks like having a branch pattern `*master*` isn't allowed
```
time="2019-12-05T22:09:18Z" level=error msg="File matches glob from unexpected ConfigMap ci-operator-master-configs instead of ci-operator-misc-configs." glob="ci-operator/config/**/*master*.yaml" source-file=ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-es6x.yaml
```

**Comment by ewolinetz** [comment]
rehearse jobs are going to fail because we don't have the supplemental `nodejs-10-rhel7` imagestream available yet

**Comment by jcantrill** [comment]
> rehearse jobs are going to fail because we don't have the supplemental `nodejs-10-rhel7` imagestream available yet

@stevekuznetsov Can we approve this without ramifications?  will this be resolved after merge? ```Eric Wolinetz  30 minutes ago
rehearse jobs aren't required for merging```

**Comment by ewolinetz** [comment]
@jcantrill rehearse jobs aren't required for merging

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -292,6 +292,12 @@ config_updater:
         default:
         - ""
       name: ci-operator-misc-configs
+    ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-feature*.yaml:
```
Comment (droslean) [inline core-services/prow/02_config/_plugins.yaml]: you don't have to do that. Please remove it

File: core-services/prow/02_config/_plugins.yaml [hunk 3]
```diff
@@ -442,6 +448,12 @@ config_updater:
         - ""
       gzip: true
       name: job-config-misc
+    ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-feature*.yaml:
```
Comment (droslean) [inline core-services/prow/02_config/_plugins.yaml]: this too

File: core-services/prow/02_config/_plugins.yaml (hunk 2 above)
Comment (ewolinetz) [reply core-services/prow/02_config/_plugins.yaml]: after debugging with @hongkailiu we found that the reason i was getting prow config failures [1] was the lack of definition in this file... this change is for a new feature branch so none of the existing patterns matched the new job and config file pattern.

[1] https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6267/pull-ci-openshift-release-master-correctly-sharded-config/10124

**Pull request merged**